### DISCUSSION TITLE
[Feature]: Return cacheable profile picture as fallback

### DISCUSSION THREAD

**Discussion by bt90** [immich-app/immich#1606] [Feature Request]
### Feature detail

Currently, if an account does not have a profile picture associated with it, the API responds with an (at least for Firefox) uncacheable 404:

![image](https://user-images.githubusercontent.com/1926175/211792508-62ce5715-f906-4d9f-bd6d-96dc9d888f03.png)

Maybe we could use HTTP 204 instead?

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/204

> A 204 response is cacheable by default

### Platform

Web

**Comment by uhthomas** [comment]
Given the response body will be zero bytes either way, I don't see the value of using 204? The browser would have to revalidate anyway and would so the behaviour would be no different.
